Makefile

```diff
@@ -89,6 +88,7 @@ check:
 	fi
 
 deploy: check stage ## to deploy staged artefacts to github pages repository
+	cp -a .circleci "$(STAGEDIR)"
```

Comment:
Can be removed?

Reply:
I have removed this in this [force push](https://github.com/hyperledger-labs/perun-doc/compare/60378e6341772a29e06cbeea7aed89ccef78a852..f3da248e6aee21b5825a847b059912427d500b45).